driver-3/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Release History
 
-## 0.14.0
+## 1.0.0-SNAPSHOT
```

Comment:
does it still need the -SNAPSHOT suffix?

Reply:
My process is to bump the version number from the develop branch.